go.mod

```diff
@@ -4,58 +4,33 @@ go 1.13
 
 require (
 	github.com/Masterminds/semver v1.5.0
-	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/go-openapi/spec v0.19.4
-	github.com/googleapis/gnostic v0.3.1
+	github.com/go-openapi/spec v0.19.8
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/onsi/ginkgo v1.10.3 // indirect
-	github.com/onsi/gomega v1.7.1 // indirect
-	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/operator-framework/operator-sdk v0.15.1
-	github.com/pkg/errors v0.8.1
-	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/cobra v0.0.5
+	github.com/openshift/api v0.0.0-20200701144905-de5b010b2b38
+	github.com/operator-framework/operator-sdk v0.18.2
+	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.5.0
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.4.0
-	github.com/uber/jaeger-client-go v2.20.1+incompatible
-	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
 	go.opentelemetry.io/otel v0.1.2
 	go.opentelemetry.io/otel/exporter/trace/jaeger v0.1.2
-	golang.org/x/net v0.0.0-20191028085509-fe3aa8a45271
-	golang.org/x/tools v0.0.0-20191205225056-3393d29bb9fe // indirect
-	google.golang.org/grpc v1.24.0
-	gopkg.in/yaml.v2 v2.2.4
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
+	google.golang.org/grpc v1.30.0
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe
-	sigs.k8s.io/controller-runtime v0.4.0
+	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
+	sigs.k8s.io/controller-runtime v0.6.0
+
+	// testing
```

Comment:
isn't this removed with `go mod tidy`?

Reply:
It is :/ I just changed the PR to remove this, and applied a `go mod tidy`.